src/content/contribute/writing-a-loader.md

```diff
@@ -110,15 +110,15 @@ The following guidelines should be followed when writing a loader. They are orde
 - Avoid __absolute paths__.
 - Use __peer dependencies__.
 
-### Simple
+### General
 
 Loaders should do only a single task. This not only makes the job of maintaining each loader easier, but also allows them to be chained for usage in more scenarios.
 
 ### Chaining
 
-Take advantage of the fact that loaders can be chained together. Instead of writing a single loader that tackles five tasks, write five simpler loaders that divide this effort. Isolating them not only keeps each individual loader simple, but may allow for them to be used for something you hadn't thought of originally.
+Take advantage of the fact that loaders can be chained together. Instead of writing one complex single loader that tackles five tasks, write five loaders that divide this effort into smaller problems. Isolating the use case for each loader keeps each one small, clear to understand, and may create an opportunity to be used for a use case you hadn't thought about from the beginning. 
 
-Take the case of rendering a template file with data specified via loader options or query parameters. It could be written as a single loader that compiles the template from source, executes it and returns a module that exports a string containing the HTML code. However, in accordance with guidelines, a simple `apply-loader` exists that can be chained with other open source loaders:
+Take the case of rendering a template file with data specified via loader options or query parameters. It could be written as a single loader that compiles the template from source, executes it and returns a module that exports a string containing the HTML code. However, in accordance with guidelines, the `apply-loader` exists that can be chained with other open source loaders:
```

Comment:
Last line of this paragraph reads really weird, not sure what to change.

Reply:
Yes, I will rewrite this and propose something clearer.